tock/employees/models.py

```diff
@@ -96,34 +96,39 @@ class UserData(models.Model):
     is_18f_employee = models.BooleanField(default=True, verbose_name='Is 18F Employee')
     is_billable = models.BooleanField(default=True, verbose_name="Is 18F Billable Employee")
     unit = models.IntegerField(null=True, choices=UNIT_CHOICES, verbose_name="Select 18F unit", blank=True)
+    float_people_id = models.CharField(max_length=50, null=True, blank=True, verbose_name='Float "people_id" attribute')
```

Comment:
Ah interesting--I agree that it makes the most sense to use whatever data type the float API is using, so `CharField` sounds good to me.  I think what the Django best practices docs are getting at, though, is that we should try normalizing to only one type of "there is no value for this field" if possible. Do you think it's OK to only use `''` as that blank value, i.e. make `None` an illegal value rather than allowing both but manually coercing all `''`'s to `None` in `save()`?  (One potential danger of relying on the latter is that in certain edge cases `save()` doesn't actually get called, which is super annoying.)

Anyways, either way this probably isn't that big a deal.  In the worst case we can do a migration at a later date to make the schema more consistent.